packages/flutter/lib/src/material/search.dart

```diff
@@ -344,6 +344,15 @@ abstract class SearchDelegate<T> {
       ..pop(result);
   }
 
+  // Closes the search page and returns to the underlying route whitout result.
+  void pop(BuildContext context) {
```

Comment:
Can this method be made private?

Reply:
Mostly so we do not expand the public API if we don't need to. If someone requests it we can make this public.

Reply:
Yes!